Sources/Views/ReloadableView.swift

```diff
@@ -120,27 +120,27 @@ extension UITableView: ReloadableView {
 
         // Update items.
         if batchUpdates.insertItems.count > 0 {
-            insertRows(at: batchUpdates.insertItems, with: .automatic)
+            insertRows(at: batchUpdates.insertItems, with: batchUpdates.animationType)
```

Comment:
Can we control the animations on the collection view in a similar way? See line 70 in this file.

Reply:
Unfortunately, in UIKit collection view doesn't have the same animation properties, you can only insert / delete / reload / move without any more possibility. :( That's why I only updated table view animations

Reply:
More than 2 months since my last answer... Could you please at least read it ?